bluesky/plans.py

```diff
@@ -2756,8 +2760,12 @@ def tune_centroid(
             "Increase num_points and/or decrease step_factor"
             " or tune_centroid will never converge to a solution"
         )
+    try:
+        motor_name, = motor.hints['fields']
+    except (AttributeError, ValueError):
+        motor_name = motor.name
     _md = {'detectors': [det.name for det in detectors],
-           'motors': [motor.name],
+           'motors': [motor_name],
```

Comment:
This one should still be `motor.name` so that it matches the device names in the descriptor!